clients/localdir/repo.go

```diff
@@ -78,10 +72,7 @@ func (r *repoLocal) AppendMetadata(m ...string) {
 
 // MakeLocalDirRepo returns an implementation of clients.Repo interface.
 func MakeLocalDirRepo(pathfn string) (clients.Repo, error) {
-	if !strings.HasPrefix(pathfn, filePrefix) {
-		return nil, fmt.Errorf("%w", errInvalidURI)
-	}
-	p := path.Clean(pathfn[len(filePrefix):])
```

Comment:
make sure to test nothin breaks after this change.

Reply:
Tested with `--local` on my workstation and it worked fine.

Reply:
Let me know if there is any other tests I can run for this.

Reply:
should be fine. Can't think of anything else atm